apps/meet-app/backend/modules/database/db_functions.bal

```diff
@@ -1,62 +1,100 @@
-// Copyright (c) 2025 WSO2 LLC. (https://www.wso2.com).
+// Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
 //
-// WSO2 LLC. licenses this file to you under the Apache License,
-// Version 2.0 (the "License"); you may not use this file except
-// in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
+// This software is the property of WSO2 LLC. and its suppliers, if any.
+// Dissemination of any information or reproduction of any material contained
+// herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+// You may not alter or remove any copyright or other notice from copies of this content.
 import ballerina/sql;
 
-# Fetch sample collections.
+# Fetch meeting types.
+#
+# + domain - meeting domain
+# + return - Meeting | Error, if not found
+public isolated function fetchMeetingTypes(string domain) returns MeetingTypes|error? {
+    record {|string domain; string types;|}|sql:Error meetingTypes = databaseClient->queryRow(
```

Comment:
Shall we create this record in the `types.bal` file inside the database module for better readability?